packages/rx_bloc_cli/test/stub.dart

```diff
@@ -28,6 +28,19 @@ final class Stub {
 
   static const licenseFileName = '{{~ licence.dart }}';
 
+  static const overwrittenLoginFiles = [
```

Comment:
@nikolay-vasilev-prime what's the purpose of this list ?

Reply:
Because there are different goldens of the login page when the social logins are enabled and disabled.